critiquebrainz/data/dump_manager.py

```diff
@@ -269,47 +282,59 @@ def importer(archive):
         try:
             with open(os.path.join(temp_dir, 'SCHEMA_SEQUENCE')) as f:
                 archive_version = f.readline()
-                if archive_version != str(model.__version__):
+                if archive_version != str(db.SCHEMA_VERSION):
                     sys.exit("Incorrect schema version! Expected: %d, got: %c."
                              "Please, get the latest version of the dump."
-                             % (model.__version__, archive_version))
+                             % (db.SCHEMA_VERSION, archive_version))
         except IOError as exception:
             if exception.errno == errno.ENOENT:
                 print("Can't find SCHEMA_SEQUENCE in the specified archive. Importing might fail.")
             else:
                 sys.exit("Failed to open SCHEMA_SEQUENCE file. Error: %s" % exception)
 
         # Importing data
-        import_data(os.path.join(temp_dir, 'cbdump', 'user_sanitised'), model.User,
-                    ('id', 'created', 'display_name', 'musicbrainz_id'))
-        import_data(os.path.join(temp_dir, 'cbdump', 'license'), model.License)
-        import_data(os.path.join(temp_dir, 'cbdump', 'review'), model.Review)
-        import_data(os.path.join(temp_dir, 'cbdump', 'revision'), model.Revision)
+        import_data(os.path.join(temp_dir, 'cbdump', 'user_sanitised'), '"user"',
```

Comment:
I don't think `'"user"'` is the best way to make sure that keywords are properly quoted. When I use `import_data` function I probably shouldn't worry about that kind of thing. What if all column names are actually quoted like it was done before?
```python
cursor.copy_from(f, '"%s"' % model.__tablename__, columns=columns)
```